sql/src/main/java/io/crate/action/job/ContextPreparer.java

```diff
@@ -205,7 +205,7 @@ private Boolean createContexts(ExecutionPhase phase, PreparerContext preparerCon
                 preparerContext.opCtx.nodeOperationMap,
                 preparerContext.leafs,
                 preparerContext.opCtx.targetToSourceMap,
-                t.getMessage()),
+                t.getClass().getSimpleName() + ": " + t.getMessage()),
```

Comment:
afaik this has nothing todo with this PR existing commits, would prefer a dedicated commit